### PR TITLE
Edu 6637 taxes endpoints for promotions & taxes api

### PR DESCRIPTION
**Pull request by julia-rabello**
#### Types of changes
- [x] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [x] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

### CODE REVIEW

File: VTEX - Promotions & Taxes API.json [hunk 1]
```diff
@@ -2333,4611 +2333,4611 @@
...
-                                            "shippingPercentualTax": {
-                                                "type": "number",
-                                                "description": "Shipping Percentual Tax over purchase total value.",
-                                                "default": 0.0
-                                            },
-                                            "percentualDiscountValueList1": {
-                                                "type": "number",
-                                                "description": "Valid discounts for the SKUs in `listSku1BuyTogether`, discount list used for Buy Together Promotions.",
-                                                "default": 0.0
-
```
Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:3497]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:3531]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:3914]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:4214]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:4225]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:4259]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:4285]: Definir `items`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:4348]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:4352]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:4429]: `default`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json:4512]: Adicionar `schema`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

Comment (karenkrieger) [inline VTEX - Promotions & Taxes API.json]: `.`

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Promotions & Taxes API.json [hunk 2]
```diff
@@ -5744,6 +5744,98 @@
...
+                                            },
+                                            "maxUsage": 0,
+                                            "idsSalesChannel": [],
+                                            "areSalesChannelIdsExclusive": false
+                                        }
+                                    ]
+                                },
+                                "schema": {
+                                    "type": "object",
+                                    "title": "",
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json:4848]: @karenkrieger não entendi essa, o schema tá declarado logo depois

File: VTEX - Promotions & Taxes API.json [hunk 3]
```diff
@@ -8072,87 +7468,696 @@
...
+                                "itemsCount": 4
+                            }
+                        }
+                    },
+                    "required": true
+                },
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "headers": {}
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json:7347]: Essa response é vazia

File: VTEX - Promotions & Taxes API.json [hunk 4]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            },
+                            "listPrice": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "validFrom": {
+                                "type": "string"
+                            },
+                            "validTo": {
+                                "type": "string"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 5]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            "price": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "listPrice": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "validFrom": {
+                                "type": "string"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 6]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "price": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "listPrice": {
+                                "type": "integer",
+                                "format": "int32"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 7]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            },
+                            "sellerId": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "price": {
+                                "type": "integer",
+                                "format": "int32"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 8]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            },
+                            "salesChannel": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "sellerId": {
+                                "type": "integer",
+                                "format": "int32"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 9]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "itemId": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "salesChannel": {
+                                "type": "integer",
+                                "format": "int32"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 10]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            },
+                            "sellerId": {
+                                "type": "string"
+                            },
+                            "validFrom": {
+                                "type": "string"
+                            },
+                            "validTo": {
+                                "type": "string"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 11]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                                "format": "int32"
+                            },
+                            "salesChannel": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "sellerId": {
+                                "type": "string"
+                            },
+                            "validFrom": {
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 12]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            "itemId": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "salesChannel": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "sellerId": {
+                                "type": "string"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 13]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "itemId": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "salesChannel": {
+                                "type": "integer",
+                                "format": "int32"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 14]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "id": {
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "itemId": {
+                                "type": "integer",
+                                "format": "int32"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 15]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            "salesChannel",
+                            "sellerId",
+                            "validFrom",
+                            "validTo"
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "id": {
+                                "type": "integer",
+                                "format": "int32"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 16]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            },
+                            "couponCode": {
+                                "type": "string"
+                            },
+                            "isArchived": {
+                                "type": "boolean"
+                            },
+                            "maxItemsPerClient": {
+                                "type": "integer",
+                                "format": "int32"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 17]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            "id",
+                            "name"
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "id": {
+                                "type": "string"
+                            },
+                            "name": {
+                                "type": "string"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Unused schema. Excluído.

[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Prices Legacy v1

File: VTEX - Promotions & Taxes API.json [hunk 18]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                        "required": [
+                            "id",
+                            "name",
+                            "quantity",
+                            "sellers"
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "id": {
+                                "type": "string"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Unused schema

File: VTEX - Promotions & Taxes API.json [hunk 19]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            "expirationIntervalPerUse"
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "utmSource": {
+                                "type": "string"
+                            },
+                            "utmCampaign": {
+                                "type": "string",
+                                "nullable": true
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Unused schema

File: VTEX - Promotions & Taxes API.json [hunk 20]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            },
+                            "couponCode": {
+                                "type": "string"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Unused schema

File: VTEX - Promotions & Taxes API.json [hunk 21]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            },
+                            "isArchived": {
+                                "type": "boolean"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Unused schema

File: VTEX - Promotions & Taxes API.json [hunk 22]
```diff
@@ -7059,1090 +8090,1088 @@
...
+                            "quantity",
+                            "sellers"
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "id": {
+                                "type": "string"
+                            },
+                            "name": {
+                                "type": "string"
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Unused schema

File: VTEX - Promotions & Taxes API.json [hunk 23]
```diff
@@ -2119,3793 +2108,4835 @@
...
+                                            "giftListTypes": [],
+                                            "productsSpecifications": [],
+                                            "affiliates": [],
+                                            "maxUsage": 0,
+                                            "maxUsagePerClient": 0,
+                                            "shouldDistributeDiscountAmongMatchedItems": false,
+                                            "multipleUsePerClient": false,
+                                            "type": "tax",
+                                            "useNewProgressiveAlgorithm": false,
+
```
[the remark this answers is not in the document]
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Acho que a resposta é só OK mesmo

File: VTEX - Promotions & Taxes API.json (hunk 1 above)
Comment (julia-rabello) [reply VTEX - Promotions & Taxes API.json]: Acho que a resposta é só OK

**Review by karenkrieger — APPROVED** [review]
LGTM

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**